docs/configure-artifact-repository.md

```diff
@@ -360,36 +360,36 @@ artifacts:
       container: my-container-name
       blob: path/in/container
       # If a managed identity has been assigned to the machines running the
-      # workflow (e.g., https://docs.microsoft.com/en-us/azure/aks/use-managed-identity)
+      # workflow (for example, https://docs.microsoft.com/en-us/azure/aks/use-managed-identity)
       # then useSDKCreds should be set to true. The accountKeySecret is not required
       # and will not be used in this case.
       useSDKCreds: true  
 ```
 
-In addition to managed identities, Argo workflows also support authentication using access keys and SAS tokens.
+You can also authenticate with [Access Keys](#using-azure-access-keys) and [Shared Access Signatures (SAS)](#using-azure-shared-access-signatures-sas).
 
-### Using Azure access keys
+### Using Azure Access Keys
 
-1. Retrieve the blob service endpoint for the storage account. For example:
+1. Retrieve the blob service endpoint for the storage account.
```

Comment:
I feel it's a complete sentence and does not require a colon at the end to introduce additional information. It would also be inconsistent with other sentences that are similar. Thanks for being ok with either way.

Reply:
> It would also be inconsistent with other sentences that are similar

Some of the sentences do use colons (e.g. https://github.com/argoproj/argo-workflows/pull/13376#discussion_r1687079403) and the docs do use colons when introducing examples in general. So having all colons would actually be more consistent. Right now it's inconsistent